<problems>
<problem>
  <file>file://$PROJECT_DIR$/bot.py</file>
  <line>123</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/bot.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Parameter 'user' unfilled</description>
  <highlighted_element>)</highlighted_element>
  <language>Python</language>
  <offset>58</offset>
  <length>1</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/moderation.py</file>
  <line>121</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/moderation.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>action_taken=&quot;warn&quot;</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>19</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/moderation.py</file>
  <line>226</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/moderation.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>action_taken=f&quot;timeout_{timeout_minutes}m&quot;</highlighted_element>
  <language>Python</language>
  <offset>20</offset>
  <length>42</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/moderation.py</file>
  <line>426</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/moderation.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>message_snapshot={&quot;content&quot;: message.content, &quot;author&quot;: str(message.author)}</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>76</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/moderation.py</file>
  <line>427</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/moderation.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>action_taken=action</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>19</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/moderation.py</file>
  <line>487</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/moderation.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>message_snapshot={&quot;content&quot;: message.content, &quot;author&quot;: str(message.author)}</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>76</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/moderation.py</file>
  <line>488</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/moderation.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>action_taken=action</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>19</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/context_menus.py</file>
  <line>407</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/context_menus.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>message_snapshot={&quot;content&quot;: self.message.content, &quot;jump_url&quot;: self.message.jump_url}</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>85</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/context_menus.py</file>
  <line>408</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/context_menus.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>action_taken=&quot;warn&quot;</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>19</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/context_menus.py</file>
  <line>522</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/context_menus.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>message_snapshot={&quot;content&quot;: self.message.content, &quot;jump_url&quot;: self.message.jump_url}</highlighted_element>
  <language>Python</language>
  <offset>20</offset>
  <length>85</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/context_menus.py</file>
  <line>523</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/context_menus.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>action_taken=f&quot;timeout_{self.duration_minutes}m&quot;</highlighted_element>
  <language>Python</language>
  <offset>20</offset>
  <length>48</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/announcements.py</file>
  <line>306</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/announcements.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>channel_id=channel.id</highlighted_element>
  <language>Python</language>
  <offset>20</offset>
  <length>21</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/announcements.py</file>
  <line>358</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/announcements.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>channel_id=channel.id</highlighted_element>
  <language>Python</language>
  <offset>24</offset>
  <length>21</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/announcements.py</file>
  <line>361</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/announcements.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>posted_at=datetime.utcnow()</highlighted_element>
  <language>Python</language>
  <offset>24</offset>
  <length>27</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>234</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>author_id=interaction.user.id</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>29</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>235</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>question=self.question</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>22</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>237</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>is_anonymous=self.is_anonymous</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>30</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>238</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>closes_at=close_time</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>20</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>405</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>option_index=option_index</highlighted_element>
  <language>Python</language>
  <offset>20</offset>
  <length>25</length>
</problem>
</problems>